src/languages/polygolf/emit.ts

```diff
@@ -228,6 +228,12 @@ export function emitExpr(
         expr.collection,
         ...emitExpr(expr.body, false, true)
       );
+    case "ForArgv":
+      return emitSexpr(
+        "for_range",
+        expr.variable,
+        expr.argcUpperBound.toString()
```

Comment:
```suggestion
        "for_argv",
        expr.variable,
        expr.argcUpperBound.toString(),
        ...emitExpr(expr.body, false, true)
```
Fixes polygolf for_argv emit. Is it fine that argcUpperBound isn't an `IR.Expr`?

Reply:
Do you have a usecase for it being `IR.Expr`? I'd say on the frontend it should definitely be an integer so it would have to be a plugin?